app/common/elasticsearchclient.py

```diff
@@ -398,8 +398,11 @@ def autocomplete(self,
             data = res['suggest'][0]['options']
         return SimpleResult(None, params, data)
 
-    def get_gene_info(self, gene_ids, facets=False, **kwargs):
+    def get_gene_info(self, gene_ids, **kwargs):
         params = SearchParams(**kwargs)
+
+        facets = kwargs.get('facets') or False
```

Comment:
the `facets` keyword is parsed and initialised in SearchParams.
```python
facets = params.facets
```
should work, or just use `params.facets` instead of `facets`